ml-agents/mlagents/trainers/stats.py

```diff
@@ -234,14 +234,14 @@ def _maybe_create_summary_writer(self, category: str) -> None:
     def _delete_all_events_files(self, directory_name: str) -> None:
         for file_name in os.listdir(directory_name):
             if file_name.startswith("events.out"):
-                logger.warning(
+                logger.debug(
```

Comment:
I'd keep this one as a warning so that users know when their old TBs are being overwritten